overlord/overlord.go

```diff
@@ -268,8 +283,8 @@ func (o *Overlord) Stop() error {
 // Settle runs first a state engine Ensure and then wait for activities to settle.
 // That's done by waiting for all managers activities to settle while
 // making sure no immediate further Ensure is scheduled. Chiefly for tests.
-// Cannot be used in conjunction with Loop.
-func (o *Overlord) Settle() error {
+// Cannot be used in conjunction with Loop. Errors if settlings takes long than timeout if timeout is not zero.
```

Comment:
Maybe “If timeout is non-zero, and settling takes longer than that timeout, returns an error”? (or at least, s/long/longer/)

Reply:
woa, that's was me at my maximum cannot write mode. sorry.